### PR TITLE
fwkissue/968: support only node >=4.3.0 for adapt-cli

### LINKED ISSUE #36: check node version in the cli to avoid version issue
Hi there, little suggestion concerning the cli, since adapt does not play well with node > 0.10.33, using `process.versions.node`` and a regex could allow a clean exit with a proper error message.

I'm not sure on how and where you could express node.js version requirements in the ecosystem, so I won't propose a patch now, but I would be glad to talk about it.

### PR DESCRIPTION
**Pull request by oliverfoster**
credits: https://github.com/gorhgorh
fixes #36

### CODE REVIEW

**Comment by gorhgorh** [comment]
yay my first PR merged, sort of :)

**Comment by moloko** [comment]
+1

**Comment by brian-learningpool** [comment]
+1, though it might be good to hold off on merging this until we can coincide with the next authoring tool release which will have the same?  Then we could put up a note in the forums.

**Comment by oliverfoster** [comment]
um, i don't mind either way. we've all got to upgrade adapt-cli anyway for it to apply.

File: bin/adapt [hunk 1]
```diff
@@ -1,3 +1,15 @@
 #!/usr/bin/env node
+var chalk = require("chalk");
+var fs = require('fs');
+var path = require('path');
+var semver = require('semver');
+var curVer = process.versions.node;
+var pkg = JSON.parse(fs.readFileSync(path.join(__dirname, '../package.json')));
+var versRange = pkg.engines.node;
```
Comment (hrajotia) [inline bin/adapt]: unused variable `versRange`!

Comment (gorhgorh) [inline bin/adapt]: good catch i intended to use it in the next if block, but it can be removed, I should have linted this :)

**Comment by hrajotia** [comment]
+1

**Comment by oliverfoster** [comment]
closed as not moved since 2016

**Comment by moloko** [comment]
is this something we'd like to resurrect now that we've got someone working on the CLI again?

**Pull request closed without merging**